### ISSUE TITLE
problem at start: basic_string::_S_create

### ISSUE THREAD

**Issue by triton13** [springlobby/springlobby#815]
![sans titre](https://user-images.githubusercontent.com/28787431/34632999-66cd8b5e-f279-11e7-9b60-a87f9520872e.png)

**Comment by supermuckl** [comment]
same for me

https://pastebin.com/YVdgf4NR

**Comment by abma** [comment]
oh, this could be a demo.

@supermuckl:

can you please try if moving the demos folder away solves the issue?

if so, can you please find out which demo causes the issue and upload it?

**Comment by abma** [comment]
please retry with springlobby 0.263 and post springlobby.log again

**Comment by triton13** [comment]
I updated, same issue


Le 8 janv. 2018 11:53 PM, "abma" <notifications@github.com> a écrit :

please retry with springlobby 0.263 and post springlobby.log again

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/springlobby/springlobby/issues/815#issuecomment-356122558>,
or mute the thread

**Comment by triton13** [comment]
I never posted springlobby log, I dont know what you mean



Le 9 janv. 2018 12:33 AM, "tristan aznar" <tristan.aznar@gmail.com> a
écrit :

I updated, same issue


Le 8 janv. 2018 11:53 PM, "abma" <notifications@github.com> a écrit :

please retry with springlobby 0.263 and post springlobby.log again

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/springlobby/springlobby/issues/815#issuecomment-356122558>,
or mute the thread

**Comment by abma** [comment]
https://springrts.com/wiki/Springlobby.log

**Comment by triton13** [comment]
2018-01-09 0:37 GMT+01:00 abma <notifications@github.com>:

**Comment by abma** [comment]
idk what you did, but attachments don't work via mail here.

can you please copy + paste the contents of springlobby.log to http://paste.springfiles.com/ please?

**Comment by triton13** [comment]
done

**Comment by abma** [comment]
please click also "create" and mail me the resulting link :-)

**Comment by triton13** [comment]
http://paste.springfiles.com/view/21cfb437

2018-01-09 1:56 GMT+01:00 abma <notifications@github.com>:

**Comment by abma** [comment]
the logfile doesn't contain the error. how did you fix it? did you receive the error message only once?

**Comment by abma** [comment]
can you paste again springlobby.log when the error happened please?

**Comment by triton13** [comment]
no i am busy sorry

**Comment by abma** [comment]
please reopen this issue / create a new issue when you can post springlobby.log with the error.

thanks!

**Comment by supermuckl** [comment]
SpringLobby 0.263 (win x32) fixed it for me - i did not deleted any demos
